src/main/resources/oas/integrations.yaml

```
@@ -0,0 +1,329 @@
+    post:
+      tags:
+        - User
+      summary: Create a new user
+      description: |
+        API to create a new user with given details
+        
+        A valid request must have atleast one of the following combination of fields in the request body -
+        - phoneNumber, phoneNumberVerified
+        - email, emailVerified
+        - username and password
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)